package.json

```diff
@@ -26,7 +26,7 @@
     "@s-ui/lint": "3",
     "@s-ui/mono": "1",
     "@s-ui/precommit": "2",
-    "@s-ui/test": "4",
+    "@s-ui/test": "4.11.0-beta.1",
```

Comment:
Temporary version until a needed `sui-test` feature is published: https://github.com/SUI-Components/sui/pull/1045